portfolio/src/main/webapp/index.html

```diff
@@ -112,90 +112,108 @@ <h3 class="section-subheading text-muted">Browse through pictures of the Black L
     <section class="page-section" id="timeline">
       <div class="container">
         <div class="text-center">
-          <h2 class="section-heading text-uppercase">History of Police Brutality</h2>
-          <h3 class="section-subheading text-muted">Where it all began.</h3>
+          <h2 class="section-heading text-uppercase">The History of Black Lives Matter</h2>
+          <h3 class="section-subheading text-muted">How it all began.</h3>
         </div>
         <ul class="timeline">
           <li>
             <div class="timeline-image">
-              <img class="rounded-circle img-fluid" src="assets/img/about/1.jpg" alt="" />
+              <img class="rounded-circle img-fluid" src="assets/img/timeline/time-1.jpg" alt="" />
             </div>
             <div class="timeline-panel">
               <div class="timeline-heading">
-                <h4>2009-2011</h4>
-                <h4 class="subheading">Our Humble Beginnings</h4>
+                <h4>2013</h4>
+                <h4 class="subheading">The Beginning</h4>
               </div>
               <div class="timeline-body">
                 <p class="text-muted">
-                  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam
-                  reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore
-                  laudantium consectetur!
+                  The death of Trayvon Martin, an African-American teenager, and the subsequent acquittal of his shooter
+                  inspires the black lives matter (blm) movement. The campaign is co-founded by three Black women:
+                  Alicia Garza, Patrisse Cullors, and Opal Tometi.
                 </p>
               </div>
             </div>
           </li>
           <li class="timeline-inverted">
             <div class="timeline-image">
-              <img class="rounded-circle img-fluid" src="assets/img/about/2.jpg" alt="" />
+              <img class="rounded-circle img-fluid" src="assets/img/timeline/time-2.jpg" alt="" />
             </div>
             <div class="timeline-panel">
               <div class="timeline-heading">
-                <h4>March 2011</h4>
-                <h4 class="subheading">An Agency is Born</h4>
+                <h4>2014/2015</h4>
```

Comment:
For covering multiple years, I'd use a - instead of /. So 2014-2015 instead of 2014/2015.

Reply:
Okay sure.